src/assets/templates/widgets/academic_plan.html

```diff
@@ -6,11 +6,7 @@ <h2 class="cc-left">Academic Plan</h2>
 
     <div class="cc-widget-padding" data-ng-if="advisorLinks.degreePlannerLink.url">
       <h3>
-        <a data-cc-campus-solutions-link-directive="advisorLinks.degreePlannerLink"
-          data-cc-campus-solutions-link-directive-cc-page-name="currentPage.name"
-          data-cc-campus-solutions-link-directive-cc-page-url="currentPage.url"
-          data-cc-campus-solutions-link-directive-cc-cache="advisingAcademics"
-          data-ng-click="expireAcademicsCache()"
+        <a ng-href="{{advisorLinks.degreePlannerLink.url}}"
```

Comment:
@marcpechaitis - Why are we changing this to a standard URL, and removing the BCS deep-link parameter support? I also notice you're taking away the cache expiring functionality. This might fix it for Advisors, but does it not then break it for Students?

Reply:
@redconfetti -- the intention behind this change is that:
1) BCS Link opens in a new window
2) Advisor _does not have the means to get back to CalCentral_ from the new window
3) The BCS backbutton is suppressed (the `ucFrom`, etc parameters area actually what causes the backbutton to appear)

This link to Degree Planner is a replacement for the now-deprecated BCS Multi-year Planner.  The old functionality needed the cache to be cleared because we _used to_ display MYP data in CalCentral, however we are not displaying data from the new Degree Planner, ergo no need to clear the cache.